src/utils/string/StringUtils.js

```diff
@@ -326,7 +326,9 @@ const getChannelListInfo = (channel, selfSignUserId, selfAnonUserId) => {
 
     channelName = helperGetMemberName(member);
 
-    channelImage = isAnonymous ? DEFAULT_PROFILE_PIC_PATH : member?.user?.image;
+    channelImage = isAnonymous
+      ? DEFAULT_PROFILE_PIC_PATH
+      : member?.user?.image || DEFAULT_PROFILE_PIC_PATH;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change is good as it adds a fallback to `DEFAULT_PROFILE_PIC_PATH` when `member?.user?.image` is undefined or null. This ensures that there's always a profile picture displayed, even if the user's image isn't available.
```diff
-    channelImage = isAnonymous ? DEFAULT_PROFILE_PIC_PATH : member?.user?.image;
+    channelImage = isAnonymous
+      ? DEFAULT_PROFILE_PIC_PATH
+      : member?.user?.image || DEFAULT_PROFILE_PIC_PATH;
```
This improves the robustness of the code by handling potential cases where the user's image might not be available.


<!-- This is an auto-generated comment by OSS CodeRabbit -->